cpu/samd21/include/periph_cpu.h

```diff
@@ -29,6 +29,13 @@ extern "C" {
 #endif
 
 /**
+ * @brief   Available core clock configurations
```

Comment:
Here you can use `@name` with opening/closing block before and after the 3 defines.